build/docker-exec

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+if [[ $# -ne 2 ]] ; then
+  echo 'build/docker-exec - run a rake command on a specified ruby version'
+  echo 'Usage:'
+  echo '  build/docker-exec 2.2.5 spec'
+  exit 1
+fi
+
+ruby_version="$1"
+rake_task="$2"
+
+cp -r /app ..
```

Comment:
Actually, what if you run bundler like this:

```bash
BUNDLE_APP_CONFIG=/tmp/.bundle/config bundle
```

It appears to be [able to control](https://github.com/bundler/bundler/blob/master/lib/bundler.rb#L180-L186) where the `.bundle` directory is being written.